identity/backend/src/test/java/io/camunda/identity/CamundaSpringBootTest.java

```diff
@@ -12,13 +12,20 @@
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.core.annotation.AliasFor;
 import org.springframework.test.annotation.DirtiesContext;
 import org.springframework.test.annotation.DirtiesContext.ClassMode;
 import org.springframework.test.context.ActiveProfiles;
 
 @SpringBootTest
 @DirtiesContext(classMode = ClassMode.BEFORE_EACH_TEST_METHOD)
-@ActiveProfiles({"test", "auth-basic", "h2"})
+@ActiveProfiles
 @Target({ElementType.TYPE})
 @Retention(RetentionPolicy.RUNTIME)
-public @interface CamundaSpringBootTest {}
+public @interface CamundaSpringBootTest {
+  @AliasFor(annotation = SpringBootTest.class, attribute = "properties")
+  String[] properties() default "";
+
+  @AliasFor(annotation = ActiveProfiles.class, attribute = "profiles")
+  String[] profiles() default {"test", "auth-basic", "h2"};
```

Comment:
Why do you use these instead of the annotations?

Reply:
With this we can have one annotation in all our test classes so they don't miss any required annotation or need to repeat it every where. So I just add these properties to be override some default setup for tests regarding attr and profiles.